Readme.md

```diff
@@ -78,14 +78,25 @@ irb(main):001:0>
 
 ## Custom Rails prompt
 
-If you want to include the current Rails environment and project name in the pry prompt, put the following lines in your project's `.pryrc`:
+If you want to include the current Rails environment and project name in the
+Pry prompt, put the following lines in your project's `.pryrc`:
 
 ```ruby
-if defined?(PryRails::RAILS_PROMPT)
-  Pry.config.prompt = PryRails::RAILS_PROMPT
+Pry.config.prompt = Pry::Prompt[:rails]
```

Comment:
`Pry.config.prompt = Pry::Prompt["rails"]` fails for me with:

> /.gem/ruby/2.5.3/gems/pry-0.12.2/lib/pry/pry_instance.rb:592:in `generate_prompt': undefined method `arity' for #<Array:0x00007fac5b16ac38> (NoMethodError)

`Pry.prompt = Pry::Prompt["rails"][:value]` works for me.

Reply:
good catch, thanks, `[:value]` will be omissible once https://github.com/pry/pry/pull/1877 is released.